client/webserver/site/src/js/mmsettings.ts

```diff
@@ -0,0 +1,1150 @@
+import {
+  PageElement,
+  BotConfig,
+  XYRange,
+  OrderPlacement,
+  BalanceType,
+  app,
+  MarketReport,
+  ConfigOption
+} from './registry'
+import { postJSON } from './http'
+import Doc from './doc'
+import BasePage from './basepage'
+import { setOptionTemplates, XYRangeHandler } from './opts'
+
+const GapStrategyMultiplier = 'multiplier'
+const GapStrategyAbsolute = 'absolute'
+const GapStrategyAbsolutePlus = 'absolute-plus'
+const GapStrategyPercent = 'percent'
+const GapStrategyPercentPlus = 'percent-plus'
+
+const driftToleranceRange: XYRange = {
+  start: {
+    label: '0%',
+    x: 0,
+    y: 0
+  },
+  end: {
+    label: '1%',
+    x: 0.01,
+    y: 1
+  },
+  xUnit: '',
+  yUnit: '%'
+}
+
+const oracleBiasRange: XYRange = {
+  start: {
+    label: '-1%',
+    x: -0.01,
+    y: -1
+  },
+  end: {
+    label: '1%',
+    x: 0.01,
+    y: 1
+  },
+  xUnit: '',
+  yUnit: '%'
+}
+
+const oracleWeightRange: XYRange = {
+  start: {
+    label: '0%',
+    x: 0,
+    y: 0
+  },
+  end: {
+    label: '100%',
+    x: 1,
+    y: 100
+  },
+  xUnit: '',
+  yUnit: '%'
+}
+
+const defaultMarketMakingConfig = {
+  gapStrategy: GapStrategyMultiplier,
```

Comment:
In `GapStrategyMultiplier`, the profit margin scales with the break-even gap, which depends on the fee rate, so this setting effectively scales the profit target with the network fee rate, which isn't the most obvious relationship. Something like `GapStrategyPercentPlus` is much more natural. Works more like the profit trigger in your arb bot, but with consideration for fees.